io.github.theforceengine.tfe.Mod.DarkForcesCommunityPatch.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "io.github.theforceengine.tfe.Mod.DarkForcesCommunityPatch",
+  "runtime": "io.github.theforceengine.tfe",
+  "sdk": "org.freedesktop.Sdk//23.08",
+  "build-options": {
+    "strip": false,
+    "no-debuginfo": true
+  },
+  "separate-locales": false,
+  "appstream-compose": false,
+  "build-extension": true,
+
+  "modules": [
+    {
+      "name": "DarkForcesCommunityPatch",
+      "sources": [
+        {
+          "type": "extra-data",
+          "url": "https://df-21.net/downloads/patches/v3/v3.zip",
+          "filename": "Dark Forces Community Patch v3.zip",
+          "sha256": "ae7eafccb503f990e78a2120756b52cd608950e49fed9342aa543913d9c845c7",
+          "size": 2415482,
+          "installed-size": 2415482
+        },
+        {
+          "type": "file",
+          "path": "io.github.theforceengine.tfe.Mod.DarkForcesCommunityPatch.metainfo.xml"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "set -e\
```

Comment:
this should be one command per line without any of these weird escape for multiline. `set -e` is no necessary.

Reply:
no. multiline escapes made this unreadable.